Author: Giuseppe Iuculano <giuseppe@iuculano.it>
Description: Only admin can upgrade wordpress. (CVE-2008-6767) (Closes: #531736)
--- a/wp-admin/upgrade.php
+++ b/wp-admin/upgrade.php
@@ -16,6 +16,8 @@ define( 'WP_INSTALLING', true );
 
 /** Load WordPress Bootstrap */
 require( '../wp-load.php' );
+if(!current_user_can('level_10'))
+	wp_safe_redirect('../wp-login.php?upgrade');
 
 timer_start();
 require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
--- a/wp-login.php
+++ b/wp-login.php
@@ -615,6 +615,8 @@ default:
 		$errors->add('registered', __('Registration complete. Please check your e-mail.'), 'message');
 	elseif	( $interim_login )
 		$errors->add('expired', __('Your session has expired. Please log-in again.'), 'message');
+	elseif  ( isset($_GET['upgrade'])) 
+		$errors->add('upgrade', __('Upgrade is needed, please log in with an admin account.'), 'message');
 
 	// Clear any stale cookies.
 	if ( $reauth )
