MU for Debian installations
--- a/wp-admin/network.php
+++ b/wp-admin/network.php
@@ -317,18 +317,13 @@ function network_step2( $errors = false
 			if ( iis7_supports_permalinks() )
 				_e( '<strong>Caution:</strong> We recommend you back up your existing <code>wp-config.php</code> file.' );
 			else
-				_e( '<strong>Caution:</strong> We recommend you back up your existing <code>wp-config.php</code> and <code>.htaccess</code> files.' );
+				printf( __( '<strong>Caution:</strong> We recommend you back up your existing <code>%s</code> and <code>/etc/wordpress/htaccess</code> files.' ), DEBIAN_FILE );
 		?></p></div>
 <?php
 	}
 ?>
 		<ol>
-			<li><p><?php
-				printf( __( 'Create a <code>blogs.dir</code> directory in <code>%s</code>. This directory is used to stored uploaded media for your additional sites and must be writeable by the web server.' ), WP_CONTENT_DIR );
-				if ( WP_CONTENT_DIR != ABSPATH . 'wp-content' )
-					echo ' <strong>' . __('Warning:') . ' ' . __( 'Networks may not be fully compatible with custom wp-content directories.' ) . '</strong';
-			?></p></li>
-			<li><p><?php printf( __( 'Add the following to your <code>wp-config.php</code> file in <code>%s</code> <strong>above</strong> the line reading <code>/* That&#8217;s all, stop editing! Happy blogging. */</code>:' ), ABSPATH ); ?></p>
+			<li><p><?php printf( __( 'Add the following to your <code>%s</code> file:' ), DEBIAN_FILE ); ?></p>
 				<textarea class="code" readonly="readonly" cols="100" rows="7">
 define( 'MULTISITE', true );
 define( 'SUBDOMAIN_INSTALL', <?php echo $subdomain_install ? 'true' : 'false'; ?> );
@@ -358,7 +353,7 @@ define( 'BLOG_ID_CURRENT_SITE', 1 );</te
 		$num_keys_salts = count( $keys_salts );
 ?>
 	<p><?php
-		echo _n( 'This unique authentication key is also missing from your <code>wp-config.php</code> file.', 'These unique authentication keys are also missing from your <code>wp-config.php</code> file.', $num_keys_salts ); ?> <?php _e( 'To make your installation more secure, you should also add:' ) ?></p>
+		printf(__( 'These unique authentication keys are also missing from your <code>%s</code> file.'), DEBIAN_FILE); ?> <?php _e( 'To make your installation more secure, you should also add:' ) ?></p>
 	<textarea class="code" readonly="readonly" cols="100" rows="<?php echo $num_keys_salts; ?>"><?php
 	foreach ( $keys_salts as $c => $v ) {
 		echo "\ndefine( '$c', '$v' );";
@@ -476,7 +471,7 @@ RewriteRule ^ - [L]';
 		$htaccess_file .= "\nRewriteRule . index.php [L]";
 
 		?>
-		<li><p><?php printf( __( 'Add the following to your <code>.htaccess</code> file in <code>%s</code>, replacing other WordPress rules:' ), ABSPATH ); ?></p>
+		<li><p><?php printf( __( 'Add the following to your <code>/etc/wordpress/htaccess</code> file, replacing other WordPress rules:' ) ); ?></p>
 		<textarea class="code" readonly="readonly" cols="100" rows="<?php echo $subdomain_install ? 11 : 16; ?>">
 <?php echo wp_htmledit_pre( $htaccess_file ); ?></textarea></li>
 		</ol>
